HelpSource/editor.css

```diff
@@ -6,7 +6,7 @@ textarea.editor {
     height: auto;
     padding: 0 1em;
     margin-top: 1em;
-    border-left: 2px solid #e1e1e1;
+    border-left: 2px solid var(--color-fg-100);
```

Comment:
also, either this border-left is `var(--color-cm-border)`, or we remove the variable cm-border, it's not used anywhere else